source/common/res/features/save-and-add-to-google-calendar/main.js

```diff
@@ -95,14 +95,41 @@
           resource: event
         });
 
-        request.execute(function (event) {
-          // appendPre('Event created: ' + name);
+        request.execute(function () {
           getSave.click();
-          alert('Your event was added to your calendar');
         });
+      }
 
-        // document.getElementById('eventName').value = '';
-        // This was reseting a form field originally??
+      /*
+       * Set RRULE's for each option in the ynab calendar select box
+       *
+       * I have yet to figure out 'TwiceAMonth'
+       */
+      function getRecur() {
+        switch (recurValue) {
+          case 'Never': return 'RRULE:FREQ=DAILY;COUNT=1';
+          case 'Daily': return 'RRULELFREQ=DAILY';
+          case 'Weekly': return 'RRULE:FREQ=WEEKLY';
+          case 'EveryOtherWeek': return 'RRULE:FREQ=WEEKLY;INTERVAL=2';
+          case 'TwiceAMonth': return '';
```

Comment:
Would this work?

```
RRULE:FREQ=MONTHLY;BYMONTHDAY=1,15
```

I suspect that's what YNAB's twice a month really is, the 1st and 15th of each month.